exercise_solutions.tex

```diff
@@ -1441,6 +1441,245 @@ \subsection*{Solution to \cref{ex:acconn}}
 But also by \cref{ex:connectivity-inductively}, each $Y(x)$ is $(-1)$-connected and all its path types are $n$-connected.
 Applying function extensionality to characterize the path types of $\prd{x:X} Y(x)$, the claim follows from the $(-1)$-connected and $n$-connected axioms of choice.
 
+
+\section*{Exercises from \cref{cha:homotopy}}
+
+\subsection*{Solution to \cref{ex:homotopy-groups-resp-prod}}
+Remember that the fundamental group of a pointed space is defined as the
+$0$-truncation of the loop space of that space.
+If we prove the equivalence without applying $\trunc 0{\blank}$ to both sides
+we are done.  We use induction on the natural numbers, with base case $1$.
+
+Suppose we have two pointed types $(A,a)$ and $(B,b)$.
+For the base case, we have $\eqv{\Omega(A \times B)}{\Omega(A) \times \Omega(B)}$
+because it is, by definition, the equivalence of pointed types:
+
+\[\eqv{ \Parens{(a,b)=(a,b), \refl{(a,b)}} }{\Parens{(a=a \times b=b), (\refl{a}, \refl{b})}}.\]
+
+By Univalence Axiom this is the same as proving the equality.
```

Comment:
I don't see where univalence comes in here?  I think this equivalence should be true without any need for univalence.  Maybe the point is that when we say "equivalence of pointed types" we mean by definition an equivalence of underlying types that respects the basepoints?